ciao-launcher/tests/examples/start_nn.yaml

```diff
@@ -3,13 +3,10 @@
 runcmd:
   - [ touch, "/etc/bootdone" ]
 start:
-  requested_resources:
-     - type: vcpus
-       value: 2
-     - type: mem_mb
-       value: 256
-     - type: network_node
-       value: 1
+  requirements:
+    vcpus: 2
+    mem_mb: 370
```

Comment:
Was there a reason for changing this value?

Reply:
Again just a copy and paste error.